### ISSUE TITLE
Reset password (no link!)

### ISSUE THREAD

**Issue by vbdigital** [jfarthing84/theme-my-login#33]
Hi,

When i click new password, i get an email but with no link to click to reset the password. :(

Can anybody help me get a link on the email?

**Comment by jfarthing84** [comment]
This is not a support forum. That is [here](https://wordpress.org/support/plugin/theme-my-login).

**Comment by alefort** [comment]
Jeff, I know you closed this issue, but I'm also experiencing the same. I've used the default email template, and even customized it with the %reset_url% and %reseturl% parameters, but the reset link is never in the email. This is my custom email template:

```
Someone requested that the password be reset for the following account:

%siteurl%

Username: %user_login%

If this was a mistake, just ignore this email and nothing will happen.

To reset your password, visit the following address:

%reset_url%
```

And the line where I have %reset_url% (and I have also tried %reseturl%) is always empty. Is there something I am missing?
